### PR TITLE
Added code review article

### PR DESCRIPTION
**Pull request by jdamerow**

### CODE REVIEW

File: content/blog/2024-10-31-code-review-paper.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Paper: Code Review in Digital Humanities"
+slug: 2024-paper-code-review
+date: 2024-10-31T09:00:15-04:00
+author: Julia Damerow, Rebecca Sutton Koeser, Malte Vogl 
+thumbnail: /images/posts/code-review-cat.png
+summary: At the DH Inside Out Workshop at DH2024, Performant Software Solutions LLC presented EditionCrafter, a tool for creating digital critical editions and archives.
```
Comment (maltevogl) [inline content/blog/2024-10-31-code-review-paper.md:7]: change summary

Comment (jdamerow) [reply content/blog/2024-10-31-code-review-paper.md:7]: 🤦‍♀️

File: content/blog/2024-10-31-code-review-paper.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Paper: Code Review in Digital Humanities"
```
Comment (rlskoeser) [inline content/blog/2024-10-31-code-review-paper.md:2]:
```suggestion
title: "Recently published: 'Code Review in Digital Humanities'"
```

File: content/blog/2024-10-31-code-review-paper.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+featureImage: /images/posts/code-review-cat.png
+featureImageAlt: A cat looking at code.
+featureImageCap: Cat "Tango" reviewing code and is not amused.
+summary: "Based on a poster at DH2022, Julia Damerow, Rebecca Sutton Koeser, Jeffrey C. Carver, and Malte Vogl have published a paper in *Digital Scholarship in the Humanities* describing the work of the Community Code Review Working Group of DHTech."
+tags:
+   - WhatsHappening
+---
+
+
+Based on a poster at DH2022, Julia Damerow, Rebecca Sutton Koeser, Jeffrey C. Carver, and Malte Vogl have published a paper in *Digital Scholarship in the Humanities* describing the work of the [Community Code Review Working Group](https://dhcodereview.github.io/) of DHTech.
```
Comment (rlskoeser) [inline content/blog/2024-10-31-code-review-paper.md:16]:
```suggestion
Expanding on a poster presented at DH2022, Julia Damerow, Rebecca Sutton Koeser, Jeffrey C. Carver, and Malte Vogl have published a paper in *Digital Scholarship in the Humanities* describing the importance of code review for humanities research software and the work of the DHTech [Community Code Review Working Group](https://dhcodereview.github.io/) to address that need. 
```

**Review by rlskoeser — APPROVED** [review]
Thank you for drafting! Made a couple of suggestions but feel free to reject them if you feel they don't improve things at all.

(I like the image 😁 🐈 💻  )

**Pull request merged**